### PR TITLE
Add a new configuration option: sentryDSN

### PR DESCRIPTION
**Pull request by mirceaulinic**
Adding a new config option, ``sentryDSN`` to provide the Sentry DSN if
the user wants to monitor the BGPalerter app for runtime exceptions.

I have updated the package.json file to the version recommended in
https://docs.sentry.io/error-reporting/quickstart/?platform=node.

This is an optional config bit that enables Sentry monitoring only when
``sentryDSN`` is provided.

This is my very first work around Node JS, not sure if that's the right
way to approach this, so I'm looking forward for your input. :-)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -22,9 +22,9 @@
       },
       "dependencies": {
         "commander": {
-          "version": "4.0.1",
```
Comment (mirceaulinic) [inline package-lock.json]: Is it expected `npm install` to generate so many diffs, or did I b0rk something?

Comment (massimocandela) [reply package-lock.json]: Don't worry about it. Please, develop against the `dev` branch in order to be sure to have the latest version of the dependencies and of the code.

**Comment by massimocandela** [comment]
I had to change branch to `dev` and some conflicts arised. 
The branch changing was needed because otherwise your feature was going to conflict with the next release of BGPalerter. 
Could you please fix these conflicts?
Can you please explain me what you would like to achieve here? (did you check the monitors directory in the dev branch? maybe it is related to this pr)

Again, thanks for your contribution.

**Comment by mirceaulinic** [comment]
Hi Massimo,

> I had to change branch to dev and some conflicts arised.

Great, haven't noticed there's a `dev` branch. I'll keep this in mind going forward, just wanted to note that you might want to consider setting `dev` as the default branch (with both `dev` and `master` being set as protected).

> Can you please explain me what you would like to achieve here? (did you check the monitors directory in the dev branch? maybe it is related to this pr)

Yes, sorry, should have clarified when opening the PR.

`monitors` help monitoring the prefixes you want to monitor, but Sentry monitors BGPalerter, the app itself. As BGPalerter runs in the background, many thing can fail without being unnoticed (or, when running inside a container, you may notice that the container has been restarted but don't have enough information on why did it fail). Sentry is a great tool to capture detailed tracebacks from where the error started, correlate with similar errors, in a visual interface, from where you can build various app reports, integration with other systems, and so on, which, ultimately helps detecting and fixing bugs.

A quick error reporting example:

Screenshot_2020-01-08 Error listen EADDRINUSE address already in use 8011

Screenshot_2020-01-08 Error listen EADDRINUSE address already in use 80112

(Perhaps not the most relevant possible example, as this one in particular would prevent the app starting at all, but I think you see where I'm pointing at).

I'll update this branch shortly.

**Comment by massimocandela** [comment]
Sorry, the `monitors` I wanted to refer to is https://github.com/nttgin/BGPalerter/tree/dev/src/uptimeMonitors I think this feature could be added there. What do you think?

**Comment by mirceaulinic** [comment]
I'm not sure they're along the same line: my understanding is that the monitors are available via the API to be pulled for various statistics. Sentry works in the different way: it watches the entire app, and when there's an exception to capture, it'll send it straight away to the Sentry server.

**Comment by massimocandela** [comment]
merge attempt, to be resolve locally

**Comment by massimocandela** [comment]
this is done!

**Comment by mirceaulinic** [comment]
Great stuff, thanks @massimocandela!

**Pull request merged**